examples/v2/cloud_functions/python/cloud_function.py

```diff
@@ -11,54 +11,106 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Creates a cloud function and calls it."""
 
+"""CloudFunction instance-type DM template.
+
+Provisions (or, creates) a pubsub topic.
```

Comment:
I thought we were not creating a pubsub topic?

Reply:
Never mind, I did it.

Reply:
Thank you :)